.github/workflows/docbuild-and-upload.yml

```diff
@@ -40,11 +40,18 @@ jobs:
       with:
         fetch-depth: 0
 
+    - name: Determine NumPy pkg-config location
+      run: |
+        echo "PKG_CONFIG_PATH=$(python -c 'import site; print(site.getsitepackages()[0])')/numpy/_core/lib/pkgconfig" >> $GITHUB_ENV
```

Comment:
I feel like sprinkling PKG_CONFIG_PATH in every workflow seems a bit hacky.
Is it possible for us to upstream something to meson/numpy to audoetect this better so that we can avoid this?

Reply:
> Is it possible for us to upstream something to meson/numpy to autodetect this better so that we can avoid this?

As far as Meson is concerned providing the `pkg_config_path` is a standard argument to setup/configure, so I don't think there is anything left that could be done in that library. For NumPy, Ralf would know best, but I am not sure that it would be able to install the pkg-config file to the normal location on a user's disk given that would be outside the typical installation architecture of a Python package

Reply:
> I feel like sprinkling PKG_CONFIG_PATH in every workflow seems a bit hacky

It's only needed for cross-compiling, so it should not be present/needed in other workflows.

With numpy 2.0, which provides `numpy-config`, this is all you need in `meson.build` (and nothing in CI jobs):
```meson
dependency('numpy')
```